src/screens/events/EventInfo.tsx

```diff
@@ -239,29 +239,46 @@ const EventInfo = ({ navigation }: EventProps) => {
+                    {hasPrivileges && !loading && (
+                        <View
+                            className={`flex-row w-full mx-4 mt-2 mb-2 ${signInPoints !== undefined && signOutPoints !== undefined ? 'justify-between' : 'justify-center'
+                                }`}
+                        >
+                            {signInPoints !== undefined && (
+                                <View className={`flex-row ${signOutPoints === undefined ? 'w-full justify-center' : 'w-[50%]'}`}>
+                                    <Octicons name="sign-in" size={24} color={darkMode ? 'white' : 'black'} />
+                                    <Text className={`ml-2 text-xl ${darkMode ? 'text-white' : 'text-black'}`}>
+                                        {signInPoints} Member
+                                    </Text>
+                                </View>
+                            )}
+
+                            {signOutPoints !== undefined && (
+                                <View className={`flex-row ${signInPoints === undefined ? 'w-full justify-center' : 'w-[50%]'}`}>
+                                    <Octicons name="sign-out" size={24} color={darkMode ? 'white' : 'black'} />
+                                    <Text className={`ml-2 text-xl ${darkMode ? 'text-white' : 'text-black'}`}>
+                                        {signOutPoints} Member
+                                    </Text>
+                                </View>
+                            )}
+                        </View>
+                    )}
+
                     {/* General Details */}
                     {nationalConventionEligible && (
-                        <Text className={`text-center mt-1 text-md ${darkMode ? 'text-grey-light' : 'text-grey-dark'}`}>
+                        <Text className={`text-center mt-1 text-md mx-5 ${darkMode ? 'text-grey-light' : 'text-grey-dark'}`}>
                             This event is eligible for national convention requirements*
                         </Text>
                     )}
 
-                    {loading && (<ActivityIndicator size="small" className='mt-3' />)}
-
-                    {(hasPrivileges && !loading) && (
-                        <View className="flex-row w-full mx-4 mt-2">
-                            <View className='flex-row w-[50%]'>
-                                <Octicons name="sign-in" size={24} color={darkMode ? "white" : "black"} />
-                                <Text className={`ml-2 text-xl ${darkMode ? 'text-white' : 'text-black'}`}>{attendanceCounts.signedInCount || 0} Member</Text>
-                            </View>
+                    {(eventType === EventType.STUDY_HOURS) && (
+                        <Text className={`mt-1 text-md mx-5 ${darkMode ? 'text-grey-light' : 'text-grey-dark'}`}>
+                            Feel free to leave the area—just be sure to scan in and out at the event location to fully earn your points*
```

Comment:
I know this is pedantic, but the dash `-` should be a period if we want this to sound professional. The asterisk also doesn't add anything to the sentence. Something more along the lines of "Feel free to leave the area. Just be sure to scan in and out at the event location to fully earn your points!"

Something like that.

Reply:
Idk. Dunno if it actually needs to be "professional sounding." That's probably just me being paranoid.

Reply:
We are a professional org, so we def do need to sound professal. Also when i mentioned the message to Daniel (Academic Officer) he mentioned something similar with the * and would be better with !